ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -442,6 +440,16 @@ objects:
             elif has_variant "large"; then
               master_type=m5.4xlarge
             fi
+            case $((RANDOM % 4)) in
+            0) AWS_REGION=us-east-1
+               ZONE_1=us-east-1b
```

Comment:
Why are we doing this?

Reply:
Seems premature, but ok.

Reply:
> Seems premature, but ok.

It is premature, but I wanted folks to be able to see the pattern without having to dig through commit/PR logs.  See "I have nothing against the other us-east-1 zones at the moment..." in the PR topic and commit message ;).